tests/ui/error-codes/E0374.stderr

```diff
@@ -1,8 +1,10 @@
-error[E0374]: the trait `CoerceUnsized` may only be implemented for a coercion between structures with one field being coerced, none found
+error[E0374]: the trait `CoerceUnsized` may only be implemented for a coercion between structures
```

Comment:
This note doesn't make sense in context -- "may only be implemented for a coercion between structures"?